### DISCUSSION TITLE
Xls Export Set Column Width

### DISCUSSION THREAD

**Discussion by ahmetkocabiyik** [filamentphp/filament#12317] [Help]
When i export xls document, columns look too narrow. Is there any method to set column width according to the width of the text or some static value.

<img width="349" alt="Screenshot 2024-05-07 at 11 50 52" src="https://github.com/filamentphp/filament/assets/8301394/f956adb3-c12e-4060-b113-41f6b4d88e36">

**Comment by ecoy1** [comment]
have u found a solution to this?

**Reply by ahmetkocabiyik** [reply]
There is no style customisation method when export xls. We use csv export it looks better than excel

**Reply by ecoy1** [reply]
thank you!!

**Comment by danharrin** [comment]
You can set column widths like this https://filamentphp.com/docs/4.x/actions/export#customizing-the-xlsx-writer
